src/game/game-reducer.js

```diff
@@ -324,5 +324,30 @@ export default function reducer(state = INITIAL_STATE, action) {
       }
     };
   }
+
+  if (action.type === actions.BOUNCE) {
+    const player = state.players[state.turn];
+    const targets = target(state, action.target, unit => {
+      return {
+        ...unit
+      };
+    });
+
+    const unitIds = Object.keys(targets);
+    return {
+      ...state,
+      players: {
+        ...state.players,
+        [state.turn]: {
+          ...player,
+          hand: [...player.hand, unitIds],
+          field: player.field.filter(
+            unitId => state.units[unitId] !== state.units[unitIds]
```

Comment:
Hmm, this doesn't make sense — `unitIds` is an array, so this second one will resolve out to like `state.units[["id-2"]]` and indexing an object by an array is no good. It works somehow, 'cause JavaScript turns it into a string for some reason?

I think all you need here is 
```
unitId => !unitIds.includes(unitId)
```